changes/3098.misc.rst

```diff
@@ -0,0 +1,2 @@
+Define Zarr-specific warning classes `ZarrDeprecationWarning` and `ZarrFutureWarning`, that
+subclass `DeprecationWarning` and `FutureWarning`, respectively.
```

Comment:
There's a few more new Warnings that should be listed here, or alternatively just point to `zarr.errors` and say there's a list in the API docs there.